students/astrawmyer/lesson09/mailroom_classes.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# new file for lesson 9 work to incorporate classes.
+
+
+class Donor:
+    def __init__(self, name, *donation):
+        self.name = name
+        self.donations = donation[0]
+
+
+    @property
+    def sum_donations(self):
+        return sum(self.donations)
+
+
+    @property
+    def avg_donations(self):
+        return sum(self.donations)/len(self.donations)
+
+    
+    def new_donation(self,new):
+        return self.donations.append(new)
+
+
+
+class Donors:
+    def __init__(self):
+        self.data = {}
+    
+
```

Comment:
Line spacing between methods in classes should be limited to one.  You can remove a lot of space here to make your class more compact.